cloud/logging.rst

```diff
@@ -26,34 +26,108 @@ The Citus Cloud dashboard automatically shows you the most recent 100 log lines
+| Hostname         | syslog.collection.us2.sumologic.com                                             |
++------------------+---------------------------------------------------------------------------------+
+| Port             | 6514                                                                            |
++------------------+---------------------------------------------------------------------------------+
+| TLS              | Yes                                                                             |
++------------------+---------------------------------------------------------------------------------+
+| Protocol         | IETF Syslog                                                                     |
++------------------+---------------------------------------------------------------------------------+
+| Message Template | <${PRI}>1 ${ISODATE} ${HOST} ${PROGRAM} ${PID} ${MSGID} [<token>@41123] $MSG\\n |
++------------------+---------------------------------------------------------------------------------+
+
+**Logentries**
+
++------------------+--------------------------------+
+| Hostname         | data.logentries.com            |
++------------------+--------------------------------+
+| Port             | 80                             |
++------------------+--------------------------------+
+| TLS              | No                             |
++------------------+--------------------------------+
+| Protocol         | IETF Syslog                    |
++------------------+--------------------------------+
+| Message Template | <token> $ISODATE $HOST $MSG\\n |
++------------------+--------------------------------+
+
+**Other**
+
+We support other providers that can receive syslog via the BSD or IETF protocols. Internally Citus Cloud uses syslog-ng, so check your providers configuration documentation for syslog-ng settings.
+
+Please reach out if you encounter any issues.
+
+External Notifications
+----------------------
+
+To monitor events in the life of a formation with outside tools via a standard format, we offer RSS feeds per organization. You can use a feed reader or RSS Slack integration (e.g. on an :code:`#ops` channel) to keep up to date.
 
-For Papertrail leave Message Template empty. Other providers might require a token to be set - follow the syslog-ng instructions for the provider, if they make them available.
+On the upper right of the "Formations" list in the Cloud console, follow the "Formation Events" link to the RSS feed. The feed includes entries for three types of events, each with the following details:
 
-3. After creation, it might take up to 5 minutes for logging to be configured. You'll then see it show up in your favorite provider's dashboard.
+**Failover Scheduled**
 
+*  Formation name
+*  Formation url
+*  Leader
+*  Failover at
 
-Supported External Providers
-----------------------------
+**Failover**
```

Comment:
Should we give some examples or explanations of these?

e.g: What does "leader", "situation" or "follower" mean here?